src/pages/questionPage.js

```diff
@@ -1,72 +1,79 @@
-'use strict';
+"use strict";
```

Comment:
Have you agreed with others that it will be used `"` instead of `'`?

Reply:
You should ideally commit this anyway in a different PR, since it makes more difficult to understand here the changes that are for the task. Also it will make more probable to have problems when merging